site/docs/components/_index/ComponentsList.module.css

```
@@ -6,6 +6,10 @@
   table-layout: fixed;
 }
 
+.componentList a svg {
+  --uitkIcon-color: var(--uitk-text-primary-foreground);
```

would you need to update the colour for interaction states also?

---

I think it would.

An easy way to achieve that would be to do:

```suggestion
  --uitkIcon-color: currentColor;
```

Then the icon will match the link's `color`, including when that changes on hover and stuff like that